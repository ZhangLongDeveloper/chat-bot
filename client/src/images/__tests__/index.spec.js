import React from 'react';
import { create } from 'react-test-renderer';

import {
  background,
  botAvatar,
  userAvatar,
} from '../index';

const mockComponent = image => (
  <img src={image} alt="test" />
);

describe('components/<App />', () => {
  it('should render background image', () => {
    const tree = create(mockComponent(background)).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('should render botAvatar image', () => {
    const tree = create(mockComponent(botAvatar)).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('should render userAvatar image', () => {
    const tree = create(mockComponent(userAvatar)).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
