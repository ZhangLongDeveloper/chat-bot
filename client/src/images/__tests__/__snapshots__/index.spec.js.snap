// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/<App /> should render background image 1`] = `
<img
  alt="test"
  src="test-file-stub"
/>
`;

exports[`components/<App /> should render botAvatar image 1`] = `
<img
  alt="test"
  src="test-file-stub"
/>
`;

exports[`components/<App /> should render userAvatar image 1`] = `
<img
  alt="test"
  src="test-file-stub"
/>
`;
